UNRELEASED.md

```diff
@@ -23,6 +23,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 - Fixed React imports in the `Filters` component to use `import * as React` for projects that don't use `esModuleInterop` ([#1820](https://github.com/Shopify/polaris-react/pull/1820))
 - Fixed `tabIndex` on `main` element causing event delegation issues ([#1821](https://github.com/Shopify/polaris-react/pull/1821))
 - Fixed icon color for destructive ActionList items ([#1836](https://github.com/Shopify/polaris-react/pull/1836))
+- Fixed not being able to explictly set `autoComplete` prop on`Autoomplete.TextField`. ([#1839](https://github.com/Shopify/polaris-react/pull/1839))
```

Comment:
```suggestion
- Fixed not being able to explictly set `autoComplete` prop on`Autocomplete.TextField`. ([#1839](https://github.com/Shopify/polaris-react/pull/1839))
```

Reply:
Haha that would have been unfortunate, thanks!